### PR TITLE
ci: fix spell format

### PR DESCRIPTION
**Pull request by lizan**
This should let the branch pass all upstream format check.

Signed-off-by: Lizan Zhou <lizan@tetrate.io>

### CODE REVIEW

**Comment by jplevyak** [comment]
I have very similar changes in the upstream PR, so we could wait for that.  @lizan what do you think the chances are to get that in in the not too distant future?  The main issues now seem to be ARM and Windows.

File: source/extensions/common/wasm/context.cc [hunk 1]
```diff
@@ -554,7 +554,7 @@ WasmResult Context::getProperty(absl::string_view path, std::string* result) {
     start = end + 1;
 
     if (first) {
-      // top-level ident
+      // top-level indent
```
Comment (jplevyak) [inline source/extensions/common/wasm/context.cc:557]: This should be "identifier".

**Comment by lizan** [comment]
@jplevyak I think it will take a while so let's merge this to avoid future break (modified CI check)

**Comment by jplevyak** [comment]
Agreed. Let's merge this and I"ll merge it into the upstream PR.

**Pull request merged**